### PR TITLE
Banner clarity enhancements

### PR DESCRIPTION
**Pull request by AndrewJLang**
Added toast for banner being enabled and some sample background images for all tiers

### CODE REVIEW

File: apps/next/pages/pricing.tsx [hunk 1]
```diff
@@ -149,7 +149,7 @@ const Page: NextPage<Props> = ({ products }) => {
                 <ModalContent>
                     <ModalHeader>
                         <Center>Almost there!</Center>
-                        <Center>Connect to Twitter to continue.</Center>
+                        <Center>Connect to all platforms to continue.</Center>
```
Comment (alexweininger) [inline apps/next/pages/pricing.tsx:152]: I think you only have to connect to Twitter for the pricing modal

File: libs/remotion/forms/src/lib/backgrounds/ImageBackground.form.tsx [hunk 2]
```diff
@@ -1,19 +1,36 @@
-import React from 'react';
-import { Box, Button, FormControl, FormHelperText, FormLabel, Input } from '@chakra-ui/react';
+import React, { useState } from 'react';
+import { Box, Button, ButtonGroup, FormControl, FormHelperText, FormLabel, Input, InputGroup, InputRightElement, Text, VStack } from '@chakra-ui/react';
 import { BackgroundComponents } from '@pulsebanner/remotion/components';
 import { LayerForm } from '../LayerForm';
 import { StarIcon } from '@chakra-ui/icons';
 
 export const ImageBackground: LayerForm<typeof BackgroundComponents.ImageBackground> = ({ props, setProps, showPricing, availableFeature }) => {
     console.log('show pricing: ', availableFeature);
```
Comment (alexweininger) [inline libs/remotion/forms/src/lib/backgrounds/ImageBackground.form.tsx:8]: Remove this

File: libs/remotion/forms/src/lib/backgrounds/ImageBackground.form.tsx [hunk 3]
```diff
@@ -1,19 +1,36 @@
...
+    const [currentString, setCurrentString] = useState(props.src ?? '');
 
     const onClickPremium = () => {
         showPricing(true);
     };
 
+    const sampleImageMap: Record<string, string> = {
+        Sky: 'http://2.bp.blogspot.com/-QkeZcIm3X_g/VJRzwWkYfXI/AAAAAAAAJWg/HhIgNTT6PtM/s1600/clouds-1500x500-twitter-header-04.jpg',
+        Rainbow: 'https://www.designbolts.com/wp-content/uploads/2014/06/Colorful-twitter-header-banner.jpg',
+        Night: 'http://1.bp.blogspot.com/-K4otZmbNpuE/U5cZmaTVi8I/AAAAAAAAHZk/PSq3iuY9zdA/s1600/twitter-header-1500x500-nature+(1).jpg',
```
Comment (alexweininger) [inline libs/remotion/forms/src/lib/backgrounds/ImageBackground.form.tsx:18]: If these images move or are removed from these urls it'll break prod

File: libs/remotion/forms/src/lib/backgrounds/ImageBackground.form.tsx [hunk 4]
```diff
@@ -1,19 +1,36 @@
...
-import { Box, Button, FormControl, FormHelperText, FormLabel, Input } from '@chakra-ui/react';
+import React, { useState } from 'react';
+import { Box, Button, ButtonGroup, FormControl, FormHelperText, FormLabel, Input, InputGroup, InputRightElement, Text, VStack } from '@chakra-ui/react';
 import { BackgroundComponents } from '@pulsebanner/remotion/components';
 import { LayerForm } from '../LayerForm';
 import { StarIcon } from '@chakra-ui/icons';
 
 export const ImageBackground: LayerForm<typeof BackgroundComponents.ImageBackground> = ({ props, setProps, showPricing, availableFeature }) => {
     console.log('show pricing: ', availableFeature);
+    const [currentString, setCurrentString] = useState(props.src ?? '');
```
Comment (alexweininger) [inline libs/remotion/forms/src/lib/backgrounds/ImageBackground.form.tsx:9]: Don't keep this state in the form, instead use the `setProps` method like I was doing before. If you don't use `setProps`, the value will never be saved to the database when the user clicks `Save settings`.

**Review by alexweininger — CHANGES_REQUESTED** [review]

File: apps/next/pages/pricing.tsx (hunk 1 above)
Comment (AndrewJLang) [reply apps/next/pages/pricing.tsx:152]: We prompt for both

Comment (AndrewJLang) [reply apps/next/pages/pricing.tsx:152]: Oh you're right for the pricing

File: libs/remotion/forms/src/lib/backgrounds/ImageBackground.form.tsx (hunk 4 above)
Comment (AndrewJLang) [reply libs/remotion/forms/src/lib/backgrounds/ImageBackground.form.tsx:9]: We do the setProps in the Set Image URL button for this

**Review by alexweininger — APPROVED** [review]

**Pull request merged**